build/commands/lib/config.js

```diff
@@ -399,10 +406,14 @@ Config.prototype.buildArgs = function () {
     delete args.brave_variations_server_url
   }
 
-  if (process.platform === 'win32') {
-    args.cc_wrapper = path.join(this.srcDir, 'brave', 'script', 'redirect-cc.cmd')
-  } else {
-    args.cc_wrapper = path.join(this.srcDir, 'brave', 'script', 'redirect-cc.py')
+  if (!this.useGoma()) {
+    // Goma is incompatible with `cc_wrapper`:
```

Comment:
Without `cc_wrapper` our  `chromium_src` directory modifications to Chromium wouldn't be applied.

Reply:
I read that as being a blocker, correct @mkarolin?

Reply:
I am unfamiliar with Goma, but unless there's another way to use our `chromium_src` files in a Goma build instead of the originals, I don't see how we'd fix those compilation errors.  So, yes, as is, I don't know that we can use this option. Perhaps @bridiver knows a way to get around the cc_wrapper || goma limitation.

Reply:
I had another look at this and found a way to use Goma with `cc_wrapper`. I can get a full build through with this patch.

Reply:
I have replicated your changes at https://github.com/brave/brave-core/pull/8252 so our CI can run on it (we don't do on forks)